tests/test_kerns.py

```diff
@@ -43,6 +43,60 @@ def test_1d(self):
             self.assertTrue(np.allclose(gram_matrix, reference_gram_matrix))
 
 
+class Matern32Old(gpflow.kernels.Stationary):
+    """
+    Old implemenetation of the Matern 3/2 kernel
+    """
+
+    @gpflow.decors.params_as_tensors
+    def K(self, X, X2=None, presliced=False):
+        if not presliced:
+            X, X2 = self._slice(X, X2)
+        r = self.scaled_euclid_dist(X, X2)
+        return self.variance * (1. + np.sqrt(3.) * r) * \
+               tf.exp(-np.sqrt(3.) * r)
+
+
+class TestMatern32(GPflowTestCase):
```

Comment:
Same here, you can remove it. And it is not advised to use GPflowTestCase. Use pytest framework.